include/boost/geometry/io/wkt/read.hpp

```diff
@@ -82,6 +82,12 @@ struct read_wkt_exception : public geometry::exception
             source += "'";
         }
         complete = message + source + " in '" + wkt.substr(0, 100) + "'";
+        // add extra diagnostic message if the problematic token included
+        // common invalid whitespace characters to help users
+        if (it->find_first_of("\t\r\n") != std::string::npos)
+        {
+			complete += "\n(note: newline and tab characters are not allowed in WKT)";
```

Comment:
Should we add line break here instead of simply appending the note to the message? We con't know exactly how this message is going to be used by the user. It not necessarily is going to be displayed using cout/cerr.